### PR TITLE
Polish style in setup.py.

### PR DESCRIPTION
**Pull request by erikrose**
* CAPITAL symbols should be reserved for constants.
* Import things more deeply to keep code itself shorter and more readable.
* There's no need to explain what a setup.py is.

(Opening this for Travis because I can't get the tests to pass locally.)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,7 +39,7 @@ def run(self):
 
 SetupTest.__doc__ = setuptools.command.test.test.__doc__
 
-EXTRA = {
+kwargs = {
```
Comment (jquast) [inline setup.py]: pep8: https://www.python.org/dev/peps/pep-0008/#constants

**Comment by jquast** [comment]
``tox -esa`` should report your style errors, https://travis-ci.org/erikrose/blessings/jobs/60041972

setup.py
  Line: 1
    pep257: D100 / Missing docstring in public module
  Line: 12
    pep257: D203 / 1 blank line required before class docstring (found 0)
  Line: 33
    pep257: D203 / 1 blank line required before class docstring (found 0)
  Line: 42
    pylint: invalid-name / Invalid constant name "kwargs"

File: setup.py (hunk 1 above)
Comment (erikrose) [inline setup.py]: Line 52 mutate it: not a constant.

Comment (jquast) [inline setup.py]: Then you should define a function, get_install_requires()

**Comment by erikrose** [comment]
I think we should remove pep257 or at least turn it way down. It's spouting nonsense, and this isn't its first violation. Remember that craziness about `__all__` needing to be a tuple? Requiring a docstring in setup.py is like requiring every module to start with a comment saying "This is a Python module".

**Comment by jquast** [comment]
I am saddened that you feel to throw away pep257 and pep8 conventions for the entire project just because you feel setup.py, or your own stylistic choices are exempt.

How does following style standards hurt? Does 'kwargs' really become more readable than 'KWARGS' or get_extra_kwargs()? does a module-level docstring hurt the readability of the project any?

(And I absolutely feel that \_\_all\_\_ should be a tuple)

**Comment by erikrose** [comment]
Consistent conventions are important—though never more important than human comprehensibilty. Remember that [a foolish consistency is the hobgoblin of little minds](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds). In particular, a docstring with no informational content is just noise—worse than no docstring at all. I'm aware that automated style checkers are all the rage at the moment, but, apart from ferreting out the most egregious offenses, they just cause arguments like this one and waste people's time. So I'm not a big fan, but I'm happy to leave them on at some low level if they make you happy.

**Comment by erikrose** [comment]
And making `__all__` a tuple does nothing. The pep257 author is not a very experienced Python programmer if he ignores the possibility that `__all__` could later be rebound.

**Comment by jquast** [comment]
I will push ignoring ``setup.py`` to prospector.yaml if that satisfies this PR.

**Comment by erikrose** [comment]
I suspect there will be more. Is one of the tools going to object if I remove the skipped line between class names and docstrings, for example at https://github.com/erikrose/blessings/blob/blessed-integration/blessings/terminal.py#L64 ? I'm literally trying to *conform* to a PEP's example here: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings. I've never seen anybody format code like that and assume one of the tools forced you into it.

**Comment by jquast** [comment]
Do whatever you want, man.

**Comment by erikrose** [comment]
I'll see if I can turn the tools down to a less zealous level. (Pointers are welcome; I don't typically use tooling.) I'm happy to meet you halfway to make this merge work, which I think is best for the users.

**Pull request merged**